src/task_manager/graph.py

```
@@ -19,6 +19,7 @@
 import common.tools
 from common import utils
 from common.graph import AgentGraph
+from common.state import Project
```

Ruff (0.11.9)

`common.state.Project` imported but unused

Remove unused import: `common.state.Project`

(F401)